features/org.jboss.tools.openshift.egit.integration.feature/.project

```diff
@@ -10,8 +10,14 @@
 			<arguments>
 			</arguments>
 		</buildCommand>
+		<buildCommand>
```

Comment:
why would you add the maven nature here? it does not harm but it wont help either?